### PR TITLE
[HackRank] Sock-Merchant

### PR DESCRIPTION
**Pull request by KartikChopra**
Added solution to SockMerchant problem in python

### CODE REVIEW

File: HackerRank Problem/Python/SockMerchant.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    while i < n-1 :
+        if ar[i] == ar[i+1]:
+            print(i)
+            count = count + 1
+            i = i + 2 
+        else:
+            i = i+1    
+
+    return count
+
```
Comment (aneels3) [inline HackerRank Problem/Python/SockMerchant.py]: Commit full code

File: HackerRank Problem/Python/SockMerchant.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    n = int(input())
+
+    ar = list(map(int, input().rstrip().split()))
+
+    result = sockMerchant(n, ar)
+
+    fptr.write(str(result) + '\n')
+
+    fptr.close()
```
Comment (aneels3) [inline HackerRank Problem/Python/SockMerchant.py]: delete the extra lines after each statement.

**Comment by KartikChopra** [comment]
Any other changes?

**Comment by aneels3** [comment]
@KartikChopra NO!

**Pull request merged**